attributes/enterprise_dashboard.rb

```diff
@@ -0,0 +1,16 @@
+# installation
+default["sensu"]["enterprise-dashboard"]["version"] = "1:1.4.0-1"
+
+# data bag
+default["sensu"]["enterprise-dashboard"]["data_bag"]["name"] = "sensu"
+default["sensu"]["enterprise-dashboard"]["data_bag"]["config_item"] = "dashboard_config"
```

Comment:
What do you prefer? `config` is already taken for Sensu itself. Maybe just `enterprise_dashboard`?